libsignal-service/src/session_store.rs

```diff
@@ -50,44 +50,36 @@ pub trait SessionStoreExt: SessionStore {
 
     // #[async_trait] sadly doesn't work here,
     // because calls to libsignal_protocol are unsend.
-    fn compute_safety_number<'s>(
+    async fn compute_safety_number<'s>(
```

Comment:
I take it that this comment is superseded, then?

Reply:
_maybe_ I need to check that it doesn't only work with `hyper` but IIRC the restriction in `libsignal` was removed.